content/docs/setup/kubernetes/quick-start-alicloud-ack/index.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Quick Start with AliCloud Kubernetes Container Service
+description: Quick Start instructions to setup the Istio service using AliCloud Kubernetes Container Service
+weight: 12
+keywords: [kubernetes,alicloud]
+---
+
+Quick Start instructions to install and run Istio in [AliCloud Kubernetes Container Service](https://cs.console.aliyun.com/) using Application Catalog.
+
+This Quick Start installs the current release version of Istio and then deploys the [Bookinfo](/docs/examples/bookinfo/) sample
+application.  It uses Application Catalog to automate the steps detailed in the [Istio on Kubernetes setup guide](/docs/setup/kubernetes/quick-start/) for Kubernetes Engine.
+
+## Prerequisites
+
+- This sample assumes that you already have an avaiable AliCloud Kubernetes cluster. Otherwise, you can create a Kubernetes cluster quickly and easily in the Container Service console.
+
+- Make sure that `kubectl` can work fine for your Kubernetes cluster.
+
+- Create a namespace to deploy Istio components, e.g. `istio-system`:
+
+{{< text bash >}}
+    $ kubectl create namespace istio-system
+{{< /text >}}
+
+- If a service account has not already been installed for Tiller, install one:
+
+{{< text bash >}}
+    $ kubectl create -f install/kubernetes/helm/helm-service-account.yaml
+{{< /text >}}
+
+- Install Tiller on your cluster with the service account:
+
+{{< text bash >}}
+    $ helm init --service-account tiller
+{{< /text >}}
+
+## Setup
+
+### Deploy Istio through Application Catalog
+
+-  Log on to the AliCloud Container Service console, and click `Application Catalog` in the left navigation pane, then select `ack-istio` in the right panel.
+
+{{< image width="100%" ratio="67.17%"
+    link="./app-catalog-istio.png"
+    caption="Istio"
+    >}}
+
+-  Customization with Parameters
```

Comment:
please again don't duplicate the helm documentation.  We have tooling to generate this table, and maintaining it in two places is more work then one :)

Reply:
+1.

Reply:
+1